Training/python/DataLoader.py

```diff
@@ -133,7 +131,7 @@ def compile_classes(file_config, file_scaling):
         _rootpath = os.path.abspath(os.path.dirname(__file__)+"/../../..")
         R.gROOT.ProcessLine(".include "+_rootpath)
 
-        _LOADPATH = "TauMLTools/Training/interface/DataLoader_main.h"
+        _LOADPATH = "TauMLTools/Training/interface/DataLoaderReco_main.h"
```

Comment:
- read path to `.h` from `.yaml`
- Would be better to have a separate `DataLoader.py` for reco (one can extract the common logic into `DataLoaderBase` module and load it)